lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -76,6 +77,10 @@ func (r *KeptnTaskReconciler) getJob(ctx context.Context, jobName string, namesp
 }
 
 func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3.KeptnTask, definition *klcv1alpha3.KeptnTaskDefinition, request ctrl.Request) (*batchv1.Job, error) {
+	serviceAccountName, errsa := getServiceAccount(definition.Spec.ServiceAccount)
+	if errsa != nil {
+		fmt.Printf("error: %v", errsa)
```

Comment:
we do not use fmt but a logger usually 😄 you can take inspo from line 108

Reply:
got it

Reply:
hey @RealAnna, when I change the `fmt.Printf("error: %v", errsa)` to `r.Log.Info("Info: ", errsa)` the tests are getting failed,
also the operator pod in klt ns are failing